docs/Dockerfile

```diff
@@ -1,6 +1,14 @@
-FROM docs/base:hugo-github-linking
+FROM docs/base:latest
 MAINTAINER Mary Anthony <mary@docker.com> (@moxiegirl)
 
+RUN svn checkout https://github.com/docker/docker/trunk/docs /docs/content/engine
```

Comment:
svn was used to checkout just the docs directory from master, not the whole repo